tests/tcp-rst-unacked-stream-10/test.yaml

```diff
@@ -0,0 +1,23 @@
+requires:
+  min-version: 8
+
+checks:
+  - filter:
+      count: 1
+      match:
+        event_type: fileinfo
+        fileinfo.sha256: 8ff57c7fc0d4babd27e2e914ad9b556b1b980a69710d3917266ec1cb4edbb782
```

Comment:
Why do we need multiple tests ? What is the difference between tcp-rst-unacked-stream-10 and tcp-rst-unacked-stream-9 ?

Reply:
they are just multiple failure cases from tlpw1, showing slightly different issues (e.g. the flow counting being off in one shows that there are multiple paths taken)

Reply:
> (e.g. the flow counting being off in one shows that there are multiple paths taken)

I do not understand this

Running suricata master on these tests I find ` stats.app_layer.flow.http: 1` for both tcp-rst-unacked-stream-09 and tcp-rst-unacked-stream-10
(so my interpretation of ` flow counting being off in one` is not correct)